be/src/storage/persistent_index.cpp

```diff
@@ -545,11 +545,10 @@ class FixedMutableIndex : public MutableIndex {
 
     size_t size() const override { return _map.size(); }
 
-    Status get(size_t n, const void* keys, IndexValue* values, KeysInfo* not_found, size_t* num_found) const override {
-        const FixedKey<KeySize>* fkeys = reinterpret_cast<const FixedKey<KeySize>*>(keys);
+    Status get(size_t n, const Slice* keys, IndexValue* values, KeysInfo* not_found, size_t* num_found) const override {
```

Comment:
how about using std::string_view 
and then you can pass it by value.

Reply:
I think it's a Slice array, not single value, so it's still passing by reference